doc/DataPlaneCodeGeneration/Handle_Ambiguity.md

```diff
@@ -0,0 +1,228 @@
+# Handle Ambiguity Between Protocol Method and Convenience Method
+
+## Background
+
+In a DPG library, protocol methods will be generated for a swagger input, and both of protocol methods and convenience methods will be generated for a typespec input. We expect to make convenience method an overload of protocol method, so that users could be very clear they are representing the same operation. For example,
```

Comment:
"TypeSpec" is a proper product name (proper noun) and should be capitalized everywhere i.e., "TypeSpec".